### DISCUSSION TITLE
Building an Electronic Voting Protocol on Fedimint

### DISCUSSION THREAD

**Discussion by elokaac** [fedimint/fedimint#1881] [General]
Hey all,

I've been following the Fedimint development for a while now. I've also been thinking about options that could be utilized to enable truly free and fair elections. With a few additional modules, fedimint seems to fit this use case very nicely.

I don't have intimate knowledge on how fedimint works, but I do have a general idea on the building blocks that make up the system.

I wrote a [document](https://aueloka.notion.site/Facilitating-Reliable-Electronic-Voting-Using-a-Distributed-Consensus-Protocol-4784375d4c3f43c5ac9ff3b3ac41234f) on Notion to share how I think fedimint can be used here.

I'd really love to hear thoughts and maybe suggestions on this.

**Comment by elsirion** [comment]
The problem with any electronic voting scheme is that, even if it was perfectly secure (to be defined), convincing the average person of that is close to impossible. Counting pieces of paper is something anyone can participate in/observe/understand, cryptographic proofs not so much (not to speak of implementations). There's a long history of debate around it, security researchers pretty much agree it's a bad idea, [linking one example article which captures some of the points here](https://medium.com/digital-diplomacy/security-experts-say-online-voting-is-a-bad-idea-heres-why-1792c9a876b0).

For DAOs and similar voluntary organizations (in contrast to coercive democracies like nation states) that might be interested in democratic elements these concerns apply less since one doesn't need to join if one doesn't understand/trust the protocols used to determine the course of action. There it would merely be a limit to the potential user base and not a potentially catastrophic problem.

**Reply by elokaac** [reply]
I think that for the US and similar democracies, the risks far outweigh the benefits. I still think that it’s something worth exploring and may be beneficial to smaller democracies, especially those who do not possess the security capabilities of the US.
